roles/osm-vector-maps/tasks/main.yml

```diff
@@ -23,11 +15,9 @@
   include_tasks: install.yml
   when: osm_vector_maps_installed is undefined
 
-
-- name: Enable/Disable/Reload NGINX for OSM, if nginx_enabled
-  include_tasks: nginx.yml
-  when: nginx_enabled | bool
-
+- name: Install OSM Vector Maps if 'osm_vector_maps_installed' not defined, e.g. in {{ iiab_state_file }}    # /etc/iiab/iiab_state.yml
+  include_tasks: install.yml
```

Comment:
FYI this is duplicate code -- Lines 14-16 (just above) runs install.yml with the standard `when:` clause (used by all other roles) -- as invoked by https://github.com/iiab/iiab/blob/master/roles/7-edu-apps/tasks/main.yml#L26-L29